src/eth_rpc/api/trace_api.rs

```diff
@@ -0,0 +1,13 @@
+use jsonrpsee::core::RpcResult as Result;
+use jsonrpsee::proc_macros::rpc;
+use reth_rpc_types::trace::parity::LocalizedTransactionTrace;
+use reth_rpc_types::BlockId;
+
+/// Trace API
+#[rpc(server, namespace = "trace")]
+#[async_trait]
+pub trait TraceApi {
```

Comment:
here we need to copy the trait and make it async?

Reply:
Mmmm I think you are thinking of the database trait? You can find it in the `tracing/database.rs` file